<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>morphing</class>
 <widget class="QMainWindow" name="morphing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>morphing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="img_source">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_dest">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>-10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browse">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_process">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>640</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browse_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>640</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_result">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>-10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>640</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>640</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Line</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
